src/playlist-maker.js

```diff
@@ -1,8 +1,16 @@
 import window from 'global/window';
-import assign from 'object.assign';
 import playItem from './play-item';
 import * as autoadvance from './auto-advance';
 
+// Lightweight Object.assign alternative.
+const assign = (target, source) => {
+  for (let key in source) {
+    if (source.hasOwnProperty(key)) {
+      target[key] = source[key];
+    }
+  }
+};
```

Comment:
would it be worth it to just use babel 6 and [babel-plugin-transform-object-assign](https://www.npmjs.com/package/babel-plugin-transform-object-assign) instead of this?